### PR TITLE
Add g:mundo_header to allow disabling the mundo header

### PR DESCRIPTION
**Pull request by nhooyr**

### CODE REVIEW

**Comment by nhooyr** [comment]
Hey @dsummersl, could you please take a look, it's a very small/simple addition.

File: doc/mundo.txt [hunk 1]
```diff
@@ -302,6 +303,13 @@ Set this to 0 to keep focus in the Mundo window after a revert.
 
 Default: 1
 
+------------------------------------------------------------------------------
+3.17 g:mundo_header                                             *mundo_header*
+
+Set this to 0 to display the mundo header.
```
Comment (simnalamburt) [inline doc/mundo.txt:309]: This is typo isn't it?

```suggestion
Set this to 0 to hide the mundo header.
```

Comment (nhooyr) [reply doc/mundo.txt:309]: Nice catch.

**Comment by simnalamburt** [comment]
@nhooyr Thanks for you contribution!

**Comment by dsummersl** [comment]
Hey @simnalamburt sorry I was indeed busy. Thanks for the contribution @nhooyr !

**Pull request merged**